styles.css

```diff
@@ -1 +1,76 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+}
+
+:root {
+    --item-color: hsl(353, 63%, 41%);
+    --bg-container: hsl(35, 95%, 91%);
+    --text-color: white;
+}
+
+.container {
+    height: 800px;
```

Comment:
Geen height attribute gebruiken in css. height en width werken wel maar zijn "deprecated"